DOCUMENTATION.md

```diff
@@ -19,7 +19,7 @@ List of subsystems shared with the host system.
 
 Name | Type | Description | `flatpak override` equivalent
 --- | --- | --- | ---
-Network | Toggle | Allow or prohibit access to the network. | `--share=network` and `--unshare=network`
+Network | Toggle | Allow or prohibit access to the network. <br /> <br /> Most programs will have this feature enabled as they require network connection. Toggling it off will prevent the program to have access to the network, therefore it won't be able to connect online. | `--share=network` and `--unshare=network`
```

Comment:
Maybe something shorter?

```
 Allow or prohibit access to the network, e.g. if disabled for Firefox, it will no longer be able to browse the internet.
```

The challenge is to find apps that will make these use examples more intuitive.

Reply:
That's a good suggestion. I was aiming more on professionalism and less to directness.

Reply:
What I think we need is more regular people language to make it useful.

Reply:
Understandable. I'm working on it and will ask for your feedback.